schema/base/land_cover.yaml

```diff
@@ -0,0 +1,50 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Land Cover Schema
+description: representation of the Earth's natural surfaces 
+type: object
+properties:
+  id:
+    "$ref": ../defs.yaml#/$defs/propertyDefinitions/id
+  geometry:
+    description: classification of surface, can correspond to the 'natural' tag from OpenStreetMap
+    unevaluatedProperties: false
+    oneOf:
+      - "$ref": https://geojson.org/schema/Polygon.json
+      - "$ref": https://geojson.org/schema/MultiPolygon.json
+  properties:
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/levelContainer
+    required:
+      - subType
+    properties:
+      subType:
+        description: type of surface represented
+        type: string
+        enum:
+          - barren
+          - crop
+          - forest
+          - grass
+          - mangrove
+          - moss
+          - shrub
+          - snow
+          - urban
+          - wetland
+      cartography:
+        suggested_min_zoom:
+          description: recommended minimum tile zoom
```

Comment:
From my limited research it's probably safe to say that these zoom levels are compatible with the Slippy Maps and Bing Maps tile standards.